jupiterone/questions/questions.yaml

```diff
@@ -0,0 +1,15 @@
+---
+sourceId: managed:template
+integrationDefinitionId: '${integration_definition_id}'
+questions: []
```

Comment:
I would pull in the `questions.yaml` from the integration project. They will be overwritten when bundling, so they should be published here or we'll lose them